styleguide/derek/incubation/homepage.ejs

```diff
@@ -1,4 +1,52 @@
 <%- partial("../pattern-components/headers/header-large-imac.ejs") %>
+<div class="rsHomepage-announcement">
+	<div class="rsHomepage-annLeft">
+		<div class="rsHomepage-annLeftContainer">
+			<div class="rsHomepage-annLeftBar">
+				<h2 class="rsHomepage-annLeftBar-header">This is an announcement H2 Here</h2>
+				<h3 class="rsHomepage-annLeftBar-subHeader">This is a sub header</h3>
+			</div>
+			<div class="rsHomepage-annLeft-content">
+				Chicharrones waistcoat squid tofu distillery single-origin coffee messenger bag letterpress green juice humblebrag.
+				Street art artisan yr, prism keffiyeh paleo pug occupy chia meditation single-origin coffee hoodie raw denim unicorn.
+				Slow-carb +1 street art, cred listicle cold-pressed banh mi raclette. Prism unicorn ramps neutra, bespoke flannel
+				pok pok crucifix health goth.
+
+				<div class="rsHomepage-annLeft-awardsRow">
+					<div class="col-sm-4">
+						<!-- Awards and Certs Pattern -->
+						<div class="awards">
+						  <h2 class="awards-title">300+</h2>
+						  <p class="awards-subhead">RedHat certifications</p>
+						  <p class="awards-subtext">including 122 CCNAs and 44 CCNPs</p>
+						</div>
+		    	</div>
+		    	<div class="col-sm-4">
+						<!-- Awards and Certs Pattern -->
+						<div class="awards">
+						  <h2 class="awards-title">300+</h2>
+						  <p class="awards-subhead">Hosting engineers across Linux, Windows and VMware</p>
+						  <p class="awards-subtext">including 122 CCNAs and 44 CCNPs</p>
+						</div>
+				  </div>
+				  <div class="col-sm-4">
+						<!-- Awards and Certs Pattern -->
+						<div class="awards">
+						  <h2 class="awards-title">300+</h2>
+						  <p class="awards-subhead">Content Management Specialists</p>
+						  <p class="awards-subtext">including 122 CCNAs and 44 CCNPs</p>
```

Comment:
Do you mind tossing in the awards partial here instead? I don't think it's super important to have the content accurate.

Reply:
@amberRrucker will do